src/components/LearnerActivityTable/index.jsx

```diff
@@ -75,14 +74,14 @@ const InactivePastMonthLearnersTable = () => {
 
   return (
     <TableContainer
-      id="inactive-past-month-learners"
-      className="inactive-past-month-learners"
-      fetchMethod={EnterpriseDataApiService.fetchInactivePastMonthLearners}
-      columns={tableColumns}
+      id={props.id}
+      className={props.id}
+      fetchMethod={() => EnterpriseDataApiService.fetchCourseEnrollments({learner_activity: props.activity})}
+      columns={getTableColumns(props)}
       formatData={formatTableData}
       tableSortable
     />
   );
 };
 
-export default InactivePastMonthLearnersTable;
+export default LearnerActivityTable;
```

Comment:
Because this component accepts the `id` and `activity` props, we should include the `PropTypes` validation for this component:

```javascript
LearnerActivityTable.propTypes = {
  id: PropTypes.string.isRequired,
  activity: PropTypes.string.isRequired,
};
```